modules/etl/camunda/element-templates/liferay-webhook-connector-boundary.json

```diff
@@ -0,0 +1,299 @@
+	},
+	"id": "io.camunda.connectors.webhook.WebhookConnectorBoundary.v1",
+	"metadata": {
+		"keywords": [
+		]
+	},
+	"name": "Webhook Boundary Event Connector",
+	"properties": [
+		{
+			"binding": {
+				"name": "inbound.type",
+				"type": "zeebe:property"
+			},
+			"type": "Hidden",
+			"value": "io.camunda:webhook:1"
+		},
+		{
+			"binding": {
+				"name": "inbound.method",
+				"type": "zeebe:property"
+			},
+			"description": "Select HTTP method.",
+			"group": "endpoint",
+			"id": "inbound.method",
+			"label": "Webhook Method",
+			"optional": false,
+			"type": "Hidden",
+			"value": "any"
+		},
+		{
+			"binding": {
+				"name": "inbound.context",
+				"type": "zeebe:property"
+			},
+			"constraints": {
+				"notEmpty": true,
+				"pattern": {
+					"message": "can only contain letters, numbers, or single underscores/hyphens and cannot begin or end with an underscore/hyphen",
+					"value": "^[a-zA-Z0-9]+([-_][a-zA-Z0-9]+)*$"
+				}
+			},
+			"description": "The webhook ID is a part of the URL.",
+			"group": "endpoint",
+			"id": "inbound.context",
+			"label": "Webhook ID",
+			"optional": false,
+			"type": "String"
+		},
+		{
+			"binding": {
+				"name": "inbound.auth.type",
+				"type": "zeebe:property"
+			},
+			"choices": [
+				{
+					"name": "None",
+					"value": "NONE"
+				},
+				{
+					"name": "Secret",
+					"value": "APIKEY"
+				}
+			],
+			"description": "Choose the authorization type.",
+			"group": "authorization",
+			"id": "inbound.auth.type",
+			"label": "Authorization Type",
+			"type": "Dropdown",
+			"value": "NONE"
+		},
+		{
+			"binding": {
+				"name": "inbound.auth.apiKey",
+				"type": "zeebe:property"
+			},
+			"condition": {
+				"equals": "APIKEY",
+				"property": "inbound.auth.type",
+				"type": "simple"
+			},
+			"description": "Expected secret.",
+			"feel": "optional",
+			"group": "authorization",
+			"id": "inbound.auth.apiKey",
+			"label": "Secret",
+			"optional": false,
+			"type": "String"
+		},
+		{
+			"binding": {
+				"name": "inbound.auth.apiKeyLocator",
+				"type": "zeebe:property"
+			},
+			"condition": {
+				"equals": "APIKEY",
+				"property": "inbound.auth.type",
+				"type": "simple"
+			},
+			"constraints": {
+				"notEmpty": true
+			},
+			"description": "A FEEL expression that extracts API key from the request. <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/liferay/#how-to-configure-secret'>See documentation</a>.",
```

Comment:
Enter a FEEL expression...

And then I would say _See the Camunda documentation._ in the `a` tag.

Reply:
It is a value already set, and it shouldn't be changed. 

I added

`A FEEL expression that extracts a secret from the request.`